utilities/cassandra/merge_operator.cc

```diff
@@ -41,6 +41,7 @@ bool CassandraValueMergeOperator::FullMergeV2(
   }
 
   RowValue merged = RowValue::Merge(std::move(row_values));
+  merged = merged.GC(gc_grace_period_in_seconds_);
```

Comment:
It's more about question: are we going to move the    purgeTtl/expireTtl logic to here as well?

Reply:
I also thought about it. I would prefer to leave ttl logic where it is now and setup some comparison later to see whether there is a performance difference.